test/fixtures/linter/rules/Directives/Directives.js

```diff
@@ -0,0 +1,97 @@
+		plugins: [ // IGNORE: Aggregation "plugins" is deprecated
+			new MultiSelectionPlugin()
+		],
+		groupBy: "some-column" // IGNORE: Association "groupBy" is deprecated
+	});
+	/* ui5linter-enable ui5-linter-no-deprecated-parameter */
+
+	includes([1], 1); // IGNORE: Function "includes" is deprecated
+
+	const getIncludesFunction = () => includes;
+	getIncludesFunction()([1], 1); // IGNORE: Function "includes" is deprecated
+	/* ui5linter-enable */
+
+	includes([1], 1); // REPORT: Function "includes" is deprecated
+
+	/* ui5linter-disable-next-line */
+	Configuration.getCompatibilityVersion("sapMDialogWithPadding"); // IGNORE: Method "getCompatibilityVersion" is deprecated
+	/* ui5linter-disable-next-line */
+	Configuration["getCompatibilityVersion"]("sapMDialogWithPadding"); // IGNORE: Method "getCompatibilityVersion" is deprecated
+
+	/* ui5linter-disable-next-line ui5-linter-no-deprecated-property */
+	Device.browser.webview; // IGNORE: "webview" is deprecated
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	Device.browser["webview"]; // IGNORE: "webview" is deprecated
+
+	Device.browser["webview"]; // REPORT: "webview" is deprecated
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	Configuration.AnimationMode; // IGNORE: Property "AnimationMode" (Enum) is deprecated
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	coreLib.MessageType; // IGNORE: Enum "MessageType" is deprecated
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property -- Followed by an intentionally Empty line
+	coreLib.MessageType; // REPORT: Enum "MessageType" is deprecated
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	let {BarColor, MessageType} = coreLib; // IGNORE: Enum "MessageType" is deprecated
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	({MessageType} = coreLib); // IGNORE: Enum "MessageType" is deprecated
+	MessageType.Error;
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	let {BarColor, MessageType: mt} = coreLib; // IGNORE: Enum "MessageType" is deprecated
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	({BarColor, MessageType: mt} = coreLib); // IGNORE: Enum "MessageType" is deprecated
+	mt.Error;
+
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property
+	mobileLib.InputType.Date; // IGNORE: Enum value "InputType.Date" is deprecated
+
+	const navigationHandler = new NavigationHandler({});
+	// ui5linter-disable-next-line ui5-linter-no-deprecated-property, ui5-linter-no-deprecated-api
+	navigationHandler.storeInnerAppState({}); // IGNORE: Method "storeInnerAppState" is deprecated
+
+
+	// ui5linter-disable ui5-linter-no-deprecated-property, ui5-linter-no-global
+	new sap.m.Button({ // IGNORE: Global variable "sap"
+		blocked: true, // IGNORE: Property "blocked" is deprecated
+		tap: () => console.log("Tapped") // IGNORE: Event "tap" is deprecated
+	});
+	// ui5linter-enable
```

Comment:
as far as I see, eslint does not support line comments for `eslint-disable` and `eslint-enable`. .... only block comments are supported. So I would recommend to have the same behavior for ui5linter comment rules

Reply:
You're right, nice catch. Also see the eslint implementation: https://github.com/eslint/eslint/blob/7fb0d957c102be499d5358a74928e0ea93913371/lib/linter/linter.js#L419

Reply:
@RandomByte as far as I see, the current implementation allows `// ui5lint-disable` which is not possible in ESLint `// eslint-disable`.